Add Pulse integration (#2658)
DataDog/integrations-extras: pulse/README.md

# Pulse

## Overview

Pulse ensures the stability and performance of your Elasticsearch and OpenSearch clusters with early issue detection, AI-generated insights, and expert support.

Pulse seamlessly integrates with Elasticsearch and OpenSearch clusters deployed anywhere, providing real-time monitoring, AI-powered analysis, and actionable recommendations to optimize resource usage and prevent downtime. For select plans, users gain direct 24/7 access to world-class Elasticsearch experts for fast, expert support, troubleshooting, upgrades, and ongoing optimization, ensuring dedicated assistance throughout their Pulse experience.

## Setup

To add Datadog as an alerting destination in Pulse, go to the **Events & Alerts** tab in you cluster's page, click **Add destinations**, and select **Datadog**.

> Make sure the integration is enabled, then choose your Datadog region (e.g. US1), and paste your **Datadog API key** into the API Key field. Optionally, you can add tags separated by commas or spaces.

Reviewer: ```suggestion
Make sure the integration is enabled, then choose your Datadog region (`US1`, for example), and paste your **Datadog API key** into the API key field. Optionally, you can add tags separated by commas or spaces.
```

> Select the alert severities you want to forward (Critical, Warning, and/or Informative), and click **Send test alert** if you want to verify it's working. The setup saves automatically - once the key is valid and at least one severity is selected, you're good to go.

Reviewer: ```suggestion
Select the alert severities you want to forward (Critical, Warning, and/or Informative), and click **Send test alert** if you want to verify it's working. The setup saves automatically - once the key is valid and at least one severity is selected, the alerts are ready to be sent.
```

## Uninstallation

To remove Datadog as an alerting destination in Pulse, go to the **Events & Alerts** tab in your cluster's page, scroll down to the Datadog card, and click the trash icon in the bottom right corner. 

> That's it - the integration will be disabled and deleted immediately, and no alerts will be sent to Datadog anymore.

Reviewer: ```suggestion
This deletes the integration and disables alerts from being sent to Datadog.
```

## Support

> If you need help with the Datadog integration or anything else in Pulse, you can open a support ticket directly through the Pulse platform, or just send us an email at **[info@pulse.support]()**.

Reviewer: ```suggestion
If you need help with the Datadog integration or anything else in Pulse, you can open a support ticket directly through the Pulse platform, or contact **[info@pulse.support]()**.
```

# Pulse

## Overview

> Pulse is a monitoring and optimization platform for Elasticsearch and OpenSearch clusters. It detects issues, tracks performance, and delivers smart alerts and recommendations to keep your search infrastructure healthy.

Reviewer: ```suggestion
Pulse is a monitoring and optimization platform for Elasticsearch and OpenSearch clusters. It detects issues, tracks performance, delivers smart alerts, and recommendations to keep your search infrastructure healthy.
```
